widget/toolbar_test.go

```diff
@@ -90,3 +90,50 @@ type toolbarLabel struct {
 func (t *toolbarLabel) ToolbarObject() fyne.CanvasObject {
 	return t.Label
 }
+
+func TestToolbarAction_Disable(t *testing.T) {
+	testIcon := theme.InfoIcon()
+	toolbarAction := NewToolbarAction(testIcon, nil)
+	object := toolbarAction.ToolbarObject().(*toolbarActionButton)
+
+	assert.False(t, object.Disabled())
+
+	toolbarAction.Disabled = true
+	object.Refresh()
```

Comment:
I don't think end users of the API have access to this call.
How will they refresh?

Reply:
I agree - can you use public APIs in this test so we can be sure it's working for end users and not just in unit tests?